<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetNameWidnow</class>
 <widget class="QDialog" name="SetNameWidnow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>113</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SetNameWidnow</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="1">
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="textLabel">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>SetNameWidnow</receiver>
   <slot>cancelButtonPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>SetNameWidnow</receiver>
   <slot>textChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>SetNameWidnow</receiver>
   <slot>okButtonPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>88</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>okButtonPressed()</slot>
  <slot>textChanged()</slot>
  <slot>cancelButtonPressed()</slot>
 </slots>
</ui>
